src/bin/club-story

```diff
@@ -200,13 +205,20 @@ const downloadFiles = (story) => {
     });
 };
 
+const createBranchFromStory = (prefix, title) => {
+    let slug = title.toLowerCase()
+        .replace(/\s/g, '-')
+        .replace(/[^a-z0-9-]/g, '')
+        .slice(0, 30);
+    debug('checking out git branch');
```

Comment:
Now that we have multiple branch types, can we add the composed branch name to this debug output?